src/test/java/AuthFailureTest.java

```diff
@@ -38,7 +29,7 @@ void clientErrorOnAuthentication() throws Exception {
                 .withPhrase("cat");
 
         final HttpClientErrorException exception = assertThrows(HttpClientErrorException.class, () -> { search.executeAsync(); } );
-        assertEquals(exception.getStatusCode(), statusCode);
+        assertEquals(statusCode, exception.getStatusCode());
```

Comment:
This and the other change to the `assertEquals()` call just below this will not change the test logic but does swap the arguments as the first is for "expected" and the second is for the "actual" value.